### ISSUE TITLE
Cannot reconfigure mapping for backslash

### ISSUE THREAD

**Issue by andreadev-it** [nvim-neo-tree/neo-tree.nvim#257]
Hello,
First of all: this is a very nice plugin, thank you for making it!
I was trying to customize the \ mapping, but it seems like it's not working.

This is the way I added it to Packer:
```
    use {
        "nvim-neo-tree/neo-tree.nvim",
        branch = "v2.x",
        requires = {
            "nvim-lua/plenary.nvim",
            "kyazdani42/nvim-web-devicons", -- not strictly required, but recommended
            "MunifTanjim/nui.nvim"
        },
        config = function()
            -- Unless you are still migrating, remove the deprecated commands from v1.x
            vim.cmd([[ let g:neo_tree_remove_legacy_commands = 1 ]])

            -- If you want icons for diagnostic errors, you'll need to define them somewhere:
            vim.fn.sign_define("DiagnosticSignError",
                {text = " ", texthl = "DiagnosticSignError"})
            vim.fn.sign_define("DiagnosticSignWarn",
                {text = " ", texthl = "DiagnosticSignWarn"})
            vim.fn.sign_define("DiagnosticSignInfo",
                {text = " ", texthl = "DiagnosticSignInfo"})
            vim.fn.sign_define("DiagnosticSignHint",
                {text = "", texthl = "DiagnosticSignHint"})
            -- NOTE: this is changed from v1.x, which used the old style of highlight groups
            -- in the form "LspDiagnosticsSignWarning"

            require("neo-tree").setup({
                window = {
                    position = "left",
                    width = 30,
                }
            })
        end
    }
```

then in my init.lua (after loading the plugins) I set this remap:
```
vim.api.nvim_set_keymap( 'n', "\\", ":Neotree toggle<cr>", {noremap = true} )
```

I saw that on the README the keymapping was inside the "config" function, at the end, but changing it there made no difference, so I tried to do it this way, but with no luck.

Basically, I just wanted to change the '\' key from "reveal" to "toggle", but it looks like nothing I do is working...

**Comment by andreadev-it** [comment]
It now seems that it started working by itself...
I had the same issue when I was trying to fix the window width. Is there something I have to do to make sure that changes in mappings/configs are actually taken into consideration?
(I always close and reopen nvim to try and see if it changes)

**Comment by cseickel** [comment]
The mapping is not anything specific to Neotree, you are just using standard built-in vim techniques to map commands. The only reason I put the mapping in the config in the quickstart is for simplicity, but you can put it anywhere at all.

> Is there something I have to do to make sure that changes in mappings/configs are actually taken into consideration?

If you are using Packer, you need to remember to source the changed file AND 'PackerCompile' or 'PackerSync' whenever you make a change to configuration.
